weaviate/gql/get.py

```diff
@@ -610,6 +619,546 @@ def with_near_image(self, content: dict, encode: bool = True) -> "GetBuilder":
         self._contains_filter = True
         return self
 
+    def with_near_audio(self, content: dict, encode: bool = True) -> "GetBuilder":
```

Comment:
I know the other filters have the same signature, but this is really hard to use.

Would it make sense to have the content of the dict as parameters? eg 
```
    def  with_near_audio(
        self,
         audio: ....
         certainty: float
        distance: float
        encode: bool = True
```

same for the other functions

Reply:
This would make the API surface different to that of the established `near_image` function, no? These changes should probably stay within the collections API development unless you'd like to get user feedback on making the arguments explicit at this stage?